src/libpriv/rpmostree-postprocess.c

```diff
@@ -254,15 +254,9 @@ do_kernel_prep (GFile         *yumroot,
       goto out;
   }
 
-  /* Copy of code from gnome-continuous; yes, we hardcode
-     the machine id for now, because distributing pre-generated
-     initramfs images with dracut/systemd at the moment
-     effectively requires this.
-     http://lists.freedesktop.org/archives/systemd-devel/2013-July/011770.html
-  */
```

Comment:
I suggest replacing this comment with another explaining what you said in the commit message.  Otherwise this empty machine ID may look a little odd when just reading the code.